Jenkinsfile

```
@@ -16,6 +16,7 @@ pipeline {
         stage('sphinx') {
           agent { label 'small' }
           steps {
+            sh 'rm -rf ./blacbox/.venv'
```

There is a typo, no?

```suggestion
            sh 'rm -rf ./blackbox/.venv'
```

---

thx!